web/src/main/resources/isoform_overrides_at_mskcc.txt

```diff
@@ -1,471 +0,0 @@
-enst_id	gene_name	refseq_id	ccds_id
```

Comment:
If this is no longer to be used, then it is correct to remove this file (and the stack which handled requests for these lists). I do wonder whether anyone in the community has need of these services though.. ?

Would it be a good idea to leave the controller in place but to simply throw an unsupportedOperation exception (or a 410 http status code) with a message such as "Our data representation for isoform overrides has been changed and this service cannot be provided without additional code refactoring. We plan to retire this service, but if you have a necessary use case for us to consider, contact the development team : (<email address>)."

Reply:
`IsoformOverrideController` is part of the internal API, so I am not sure if it is worth the effort. I am not even sure if this endpoint has ever been used somewhere.

Reply:
Ok, that sounds good. If it was not visible as part of the public API then definitely not worth it.